Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -220,7 +220,7 @@ static Boolean __CGPathEqual(CFTypeRef cf1, CFTypeRef cf2) {
 }
 
 /**
- @Status Interoperable
+@Status Interoperable
```

Comment:
This kind of stuff can cause annoying merge conflicts :)

---
In reply to: [91582171](https://github.com/Microsoft/WinObjC/pull/1507#discussion_r91582171) [](ancestors = 91582171,90973712)